src/resources/views/finances/modals/create_operation.blade.php

```diff
@@ -4,31 +4,33 @@
     <span class="close-modal"><i class="bi bi-x"></i></span>
     <h2>Pridať operáciu</h2>
     <div>
-      <input class="operation_type" id="income_choice" name="operation_type" type="radio" value="income">
+      <input class="operation_type" id="income_choice" name="operation_type" type="radio" value="income" checked>
       <label for="income_choice"><i>Príjem</i></label>
       <input class="operation_type" id="expense_choice" name="operation_type" type="radio" value="expense">
       <label for="expense_choice"><i>Výdavok</i></label>
     </div>
     <select id="operation_choice" name="typ">
       <option value="default_opt">Vyberte typ operácie</option>
 
-      <option class="expense_opt" value="">Náklady na služobnú cestu</option>
-      <option class="expense_opt" value="">Malý nákup</option>
-      <option class="expense_opt" value="">Nákup na faktúru</option>
-      <option class="expense_opt" value="">Nákup z Marquetu</option>
-      <option class="expense_opt" value="">Pôžička pre niekoho</option>
+      <option class="expense_opt" value="" style="display: none;">Náklady na služobnú cestu</option>
+      <option class="expense_opt" value="" style="display: none;">Malý nákup</option>
+      <option class="expense_opt" value="" style="display: none;">Nákup na faktúru</option>
+      <option class="expense_opt" value="" style="display: none;">Nákup z Marquetu</option>
```

Comment:
you should create new class, not add so much styles to html code

Reply:
done